Castle.Windsor-SL.sln

```diff
@@ -1,68 +0,0 @@
-﻿
```

Comment:
All the Silverlight changes merged in the other PR are still showing up here, maybe you just need to rebase to get GitHub to refresh the diff.